cabal.project.wasm32-wasi

```diff
@@ -1,6 +1,9 @@
 packages: .
 
 package unix
+  -- Necessary as otherwise the `allow-newer` below will cause cabal to fail to
+  -- infer a consistent value of this flag.
+  flags: +os-string
```

Comment:
Please do as the other comment says. Remove the filepath from `allow-newer`.

Reply:
Done.